demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -46,13 +42,16 @@ def build_arg_parser():
                       default=100, type=int)
     args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask. Ignored in GUI mode",
                       default=20, type=int)
-    args.add_argument("-mc", "--mask_color",
-                      help="Optional. Color to be treated as mask (provide 3 RGB components in range of 0...255)."
-                      " Default is 0 0 0. Ignored in GUI mode", default=[0, 0, 0], type=int, nargs=3)
     args.add_argument("--no_show", help="Optional. Don't show output. Cannot be used in GUI mode", action='store_true')
     args.add_argument("-o", "--output", help="Optional. Save output to the file with provided filename."
                       " Ignored in GUI mode", default="", type=str)
-    args.add_argument("-a", "--auto", help="Optional. Use automatic (non-interactive) mode instead of GUI",
+    args.add_argument("-am", "--auto_mask", help="Optional. Use automatic (non-interactive) mode with color mask."
+                      "Provide color to be treated as mask (3 RGB components in range of 0...255). "
+                      "Cannot be used together with -ar.",
+                      default=None, type=int, nargs=3)
```

Comment:
I would suggest `metavar='C'` or something similar, otherwise the usage message looks weird.

Reply:
Done